Add post for 0.14.2 release

TobiGr opened the pull request:
- [ ] Check localization information @theScrabi 

- [x] Correct release date in YAML header

- [x] Add screenshot for linear downloads list

- [x] Add info about newly added languages

@theScrabi What about https://github.com/TeamNewPipe/NewPipe/pull/1763 and https://github.com/TeamNewPipe/NewPipe/pull/1628? Are you going to include them into this release?

CC @TheAssassin ping :D

TobiGr commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+
+[@coin3x](https://github.com/coin3x) fixed the video queuing behaviour. When the latest queue already finished playing, enqueuing a new video or audio will start the stream by its own.
+
+[@theScrabi](https://github.com/theScrabi) worked on improving NewPipe's localization handling. For devices where the content country and the actual location differ, the number of views and video descriptions are now displayed in the correct language. 
```
@theScrabi I am pretty sure this paragraph contains false information about your localization changes. What exactly did you fix in https://github.com/TeamNewPipe/NewPipe/pull/1792?

TheAssassin replied:
This should indeed be fact checked.

theScrabi commented:
Can you please also add an issue about the 0.14.2 release. Please put the features from the release draft in it, ill put the build version in and start the release fork.

TobiGr commented:
> Can you please also add an issue about the 0.14.2 release

@theScrabi You opened the merge request. Is it still necessary to open an issue for the next release? What do you need it for? Collecting things that need to be fixed or tracking something else?

theScrabi commented:
No, i was just faster :D

theScrabi commented:
The pr/issue are used to discuss the upcomminf version.

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+excerpt_separator: <!-- more -->
+---
+
+We have just released NewPipe 0.14.2. Such as NewPipe 0.14.0 this version mostly brings UI and UX improvements
```
"Such as" is the wrong wording here.

```suggestion
We have just released NewPipe 0.14.2. Just like NewPipe 0.14.0 this version mostly brings UI and UX improvements
```

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+---
+
+We have just released NewPipe 0.14.2. Such as NewPipe 0.14.0 this version mostly brings UI and UX improvements
+which include new layouts for tablets and an additional linear layout in "Downloads".
```
```suggestion
including new layouts for tablets and an additional linear layout in "Downloads".
```

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+We have just released NewPipe 0.14.2. Such as NewPipe 0.14.0 this version mostly brings UI and UX improvements
+which include new layouts for tablets and an additional linear layout in "Downloads".
+
+Furthermore, the subscriptions page got a small upgrade, several bugs were fixed and new translations added.
```
Try to avoid "get" if possible. I'd further split the sentence into two parts.

```suggestion
Furthermore, the subscriptions page receievd a small upgrade. Also, several bugs were fixed and new translations were added.
```

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+
+## New tablet UI
+
+[@nv95](https://github.com/nv95) brings a completely new feeling to NewPipe on tablets: he implemented grid layouts for video lists like the search, a channel's videos or the trending page. He took a look at the subscriptions page, too:
```
```suggestion
[@nv95](https://github.com/nv95) brings a completely new feeling to NewPipe on tablets. He implemented grid layouts for video lists like for example the search, a channel's videos or the trending page. He took a look at the subscriptions page, too:
```

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+
+<img class="no-flow vertical" src="/img/screenshots/shot_subscriptions_tablet_dark.png"/>
+
+On top of that, he also made the video detail view responsive. This means, next and recommended videos will be displayed at the right side of your screen if your device is large enough.
```
No comma here.

```suggestion
On top of that, he also made the video detail view responsive. From now on, next and recommended videos will be displayed at the right side of your screen if your screen's resolution is large enough.
```

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+
+## Improved subscription handling
+
+This version brings also interesting changes to the subscription page. [@somethingwieredhere](https://github.com/somethingweiredhere) added a new longpress menu to it. From within the menu you can share and delete a subscription. This is helpful whenever a channel was deleted or renamed by YouTube. So if you have had dead channels before, it is now time to clean them up!
```
```suggestion
This version brings also interesting changes to the subscription page. [@somethingwieredhere](https://github.com/somethingweiredhere) added a new longpress menu to it. From within the menu you can share and delete a subscription. This can be really useful whenever a channel was deleted or renamed by YouTube. So, if you have had dead channels before, it is now time to clean them up!
```

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+
+## Further changes
+
+[@ritiek](https://github.com/ritiek) added separate gesture controls to the player. This means, you can disable either the brightness or volume gesture and still use the other one. You can turn both of, of course.
```
Avoid "This means", it's redundant in this context.
```suggestion
[@ritiek](https://github.com/ritiek) added separate gesture controls to the player. You can disable either the brightness or volume gesture and still use the other one. You can turn both of, of course.
```

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+
+[@ritiek](https://github.com/ritiek) added separate gesture controls to the player. This means, you can disable either the brightness or volume gesture and still use the other one. You can turn both of, of course.
+
+[@MaX-Lo](https://github.com/MaX-Lo) made the player remember the last aspect ratio. In case you have a 18:9 device or just like to view video in another than the default aspect ratio, you will definitely enjoy this improvement. 
```
Is 18:9 really a thing?

TobiGr replied:
Yes. See https://github.com/TeamNewPipe/NewPipe/issues/1318#issuecomment-382398636.
You can find several devices using a search engine of your choice :P
Here is just a small collection: Samsung Galaxy S8, LG G6, Huawei Nova 2 / P10, Sony Xperia XZ1 Compact. Even Apple use this ratio for the IPhone 8 and SE.

This is a significant enhancement for 2:1 devices as well. So if you are more comfortable with 2:1, we can use this instead.

TheAssassin replied:
I wonder why they call it 18:9 instead of 2:1... Thanks for the heads-up. You should add a link to that issue!

TobiGr replied:
If I add a link to this issue, I should add links to most other addressed issues, too. Just for consistency. I guess there will be some more changes before we release the new version. With other words, this can wait till tomorrow

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+[@theScrabi](https://github.com/theScrabi) worked on improving NewPipe's localization handling. For devices where the content country and the actual location differ, the number of views and video descriptions are now displayed in the correct language. 
+He also added a new content setting which allows you to set the content language. 
+
+Speaking of localization, we would like to thank all translators who updated NewPipe and added support for two new languages: (Don't remember which, will insert here once Weblate was merged. Taking a look into the commit summary is less time expensive then scrolling through the history) All together, NewPipe comes with translations for nearly [55 languages](https://hosted.weblate.org/projects/new ...
```
Big, fat ***FIXME*** in that line. Please fix before merge.

TheAssassin replied:
"All together" should be "Altogether", shouldn't it?

TobiGr replied:
> Big, fat FIXME in that line. Please fix before merge.

I know that this information misses. That's why I added the ToDo list above. I need to wait for @theScrabi to merge WebLate.

>"All together" should be "Altogether", shouldn't it?

yes.

TheAssassin commented on _posts/2018-10-16-newpipe-0.14.2-released.md:
```diff
@@ -0,0 +1,58 @@
+
+[@BO41](https://github.com/BO41) cleaned up the code which results in a smaller app size and tiny performance improvements.
+
+[@theScrabi](https://github.com/theScrabi) took a look at another bug in this version: after fixing time parsing for formats like "02.56", people can use NewPipe in Finland now.
```
I'd add "... and other countries with strange date formatting". Show the Finnish how weird their language how weird their language is!

TobiGr replied:
Hm. I wouldn't do that :D

TobiGr commented:
> I only found a few minor things, and added suggestions using that new GitHub feature thingy. Please let me know how well that works.

@TheAssassin This seems to be quite useful. But we definitely have to squash the commits. Otherwise this feature will result in a messy commit history...

TobiGr commented:
@theScrabi I've done everything what I can do, Please check the localization information. The release date is set to today. In case, you do not release the new version today, we need to change it.

Before we merge this PR, we should squash the commits.

TobiGr commented:
Okay, squashed and added information about the Android 9 player switching bug:

> Despite the work @theScrabi and boredomdenied put into this, we are aware of a bug which resets the playback position when switching between players. Due to the fact that bigger changes in the player code are needed, we aim to fix this and other minor Android 9 related bugs in the next release.

Is the information about the localization correct?

I'll post a small statement to the "This Week In F-Droid" submission thread once this post is online.

theScrabi commented:
:)

TheAssassin commented:
Didn't see this earlier, deploying right now, sorry for the delay.
